client/src/components/App/__snapshots__/App.test.jsx.snap

```diff
@@ -0,0 +1,375 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<App /> should render correctly 1`] = `
+ShallowWrapper {
+  "length": 1,
+  Symbol(enzyme.__root__): [Circular],
+  Symbol(enzyme.__unrendered__): <App />,
+  Symbol(enzyme.__renderer__): Object {
+    "batchedUpdates": [Function],
+    "getNode": [Function],
+    "render": [Function],
+    "simulateEvent": [Function],
+    "unmount": [Function],
+  },
+  Symbol(enzyme.__node__): Object {
+    "instance": null,
+    "key": undefined,
+    "nodeType": "host",
+    "props": Object {
+      "children": Array [
+        <Icons
+          toggleCarousel={[Function]}
+        />,
+        <div
+          id="showCarousel1"
+          onClick={[Function]}
+        />,
+        null,
+        null,
+        <div
+          id="breadcrumbs"
+        >
+           
+          Home / 
+          
+           / 
+          
+        </div>,
+        <div
+          id="hostelName"
+        >
+           
+          
+           
+        </div>,
+        <div
+          id="location"
+        >
+          <i
+            className="fas fa-map-marker-alt"
+          />
+          
+          , 
+          
+          ,
```

Comment:
There seems to be a lot of extra spacing in this file. Is this necessary?

Reply:
it is from the snapshot from using my test. I didn't actually make this file, so I am not sure.